react/features/settings/functions.js

```diff
@@ -92,6 +114,7 @@ export function getMoreTabProps(stateful: Object | Function) {
     const framerate = state['features/screen-share'].captureFrameRate ?? SS_DEFAULT_FRAME_RATE;
     const language = i18next.language || DEFAULT_LANGUAGE;
     const configuredTabs = interfaceConfig.SETTINGS_SECTIONS || [];
+    const notifications = getNotificationsMap(stateful);
```

Comment:
would rename this to `activeNotifications` or `enabledNotifications` so we don't mistake this with the config's `notifications` array